docs/tutorials/concrete-ml/concrete-ml.ipynb

```diff
@@ -91,8 +91,8 @@
     "from concrete.ml.torch.compile import compile_torch_model\n",
     "from torch.nn.modules.loss import _Loss\n",
     "import torch.nn as nn\n",
-    "from fedbiomed.researcher.environ import environ\n",
-    "tensorboard_dir = environ['TENSORBOARD_RESULTS_DIR']"
+    "from fedbiomed.researcher.config import config\n",
```

Comment:
@srcansiz some of the examples in `docs/tutorials` and `notebooks` don't have the 
`from fedbiomed.researcher.config import config` line.

Probably we should add it ?

Reply:
Hi @mvesin,
It is normal they don't have if it is not used ?

Reply:
> It is normal they don't have if it is not used ?

OK my bad, I thought we always needed to do an `from fedbiomed.researcher.config import config` in the researcher. But I confirm that this is correctly done when doing `exp = Experiment(...)` including in the case we use a non default researcher config file.

So we just want to remove the `from fedbiomed.researcher.config import config` from `notebooks/101_getting_startes.ipynb` where it is imported and not used. This may mislead the user into thinking you always need to import config.